code/modules/surgery/organs/autosurgeon.dm

```diff
@@ -149,6 +149,10 @@
 	surgery_speed = 0.75
 	loaded_overlay = "autosurgeon_syndicate_loaded_overlay"
 
+/obj/item/autosurgeon/syndicate/Initialize(mapload)
+	. = ..()
+	ADD_TRAIT(src, TRAIT_CONTRABAND, TRAIT_GENERIC)
```

Comment:
`INNATE_TRAIT`

Reply:
For all of these: Since they're being applied in `Initialize`, we like to use `INNATE_TRAIT`, especially since `TRAIT_GENERIC` is used in a few other places which we might overlap